### ISSUE TITLE
Unable to create docker networks

### ISSUE THREAD

**Issue by linushex** [mullvad/mullvadvpn-app#836]
# Description

- Debian/KDE
- Mullvad App 2019.3 (local network sharing enabled)
- Docker 18.09.1, build 4c52b90

When creating a docker network with `docker network create test`, docker displays:
```
Error response from daemon: could not find an available, non-overlapping IPv4 address pool among the defaults to assign to the network
```

Similarly, when running `docker-compose up`:
```
Creating network "project_default" with the default driver
ERROR: could not find an available, non-overlapping IPv4 address pool among the defaults to assign to the network
```
**Output of `docker network ls`**:
```
NETWORK ID          NAME                DRIVER              SCOPE
479dd90f5e93        bridge              bridge              local
642b7f369e6d        host                host                local
10a17f7b0013        none                null                local
```

**Output of `ip addr`**:
```
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever
2: enp0s25: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
    link/ether bb:de:8b:9d:aa:61 brd ff:ff:ff:ff:ff:ff
    inet 192.168.0.223/24 brd 192.168.0.255 scope global dynamic noprefixroute enp0s25
       valid_lft 3473sec preferred_lft 3473sec
3: wlp2s0: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000
    link/ether 1f:96:3a:fb:53:6f brd ff:ff:ff:ff:ff:ff
4: docker0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN group default 
    link/ether bf:02:1f:17:d9:a6 brd ff:ff:ff:ff:ff:ff
    inet 172.17.0.1/16 brd 172.17.255.255 scope global docker0
       valid_lft forever preferred_lft forever
5: tun0: <POINTOPOINT,MULTICAST,NOARP,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UNKNOWN group default qlen 100
    link/none 
    inet 10.15.0.4/16 brd 10.15.255.255 scope global tun0
       valid_lft forever preferred_lft forever
    inet6 fdda:d0d0:cafe:1301::1002/64 scope global 
       valid_lft forever preferred_lft forever
    inet6 fe80::6f44:7fe1:edb7:4c93/64 scope link stable-privacy 
       valid_lft forever preferred_lft forever
```

I also get the following extra routes in **`ip route`**:
```
0.0.0.0/1 via 10.14.0.1 dev tun0
...default
10.14.0.0/16 dev tun0 proto kernel scope link src 10.14.0.4 
128.0.0.0/1 via 10.14.0.1 dev tun0 
141.98.252.134 via 192.168.0.1 dev enp0s25
...
...
...
```

Where can I check that the local network sharing is really activated ?

If I quit the Mullvad App, create the network, it stays created and I can reactivate the App afterwards.

There is no problem using config files and `network-manager` and its OpenVPN plugin. I only experienced this problem when I started using the Mullvad App. In fact, these are the extra routes I get when using `network-manager`:
```
default via 10.11.0.1 dev tun0 proto static metric 50 
...default
10.11.0.0/16 dev tun0 proto kernel scope link src 10.11.0.11 metric 50 
...
...
185.178.48.114 via 192.168.0.1 dev enp0s25 proto static metric 100 
...
...
```

There is no problem in macOS.

Related issues: 

https://github.com/mullvad/mullvadvpn-app/issues/529

**Comment by hovancik** [comment]
Same issue here.

**Comment by egasimus** [comment]
Experiencing the same issue on Linux. Also Docker port bindings don't work.

**Comment by webyneter** [comment]
Same for me.

**Comment by webyneter** [comment]
This solved the issue for me:

> Did you enable local network sharing in the app? (Settings > Preferences)
https://github.com/mullvad/mullvadvpn-app/issues/529#issuecomment-429963983

**Comment by Yumekui** [comment]
> This solved the issue for me:
> 
> > Did you enable local network sharing in the app? (Settings > Preferences)
> > [#529 (comment)](https://github.com/mullvad/mullvadvpn-app/issues/529#issuecomment-429963983)

Still doesn't work for me. I used the cli to activate that setting though.
`mullvad lan set allow`

**Comment by hovancik** [comment]
Hey guys,

this seems to help me, adding to `compose` file: 

```
network_mode: "bridge"
```

source:  https://stackoverflow.com/questions/43720339/docker-error-could-not-find-an-available-non-overlapping-ipv4-address-pool-am/57152380#57152380

Will have to test more, but first container seems to work fine. 

In past I used another solution from the same thread and was adding custom network.

**Comment by dlon** [comment]
Is anyone still having this issue? I'll close but please reopen if you do.
